### ISSUE TITLE
Unable to find resource id in android build on UI Automator even after giving accessibilityLabel.

### ISSUE THREAD

**Issue by arughani** [appium/appium#13232]
## The problem

Unable to find resource id in android build on UI Automator even after giving accessibilityLabel on release build of android.

## Environment
*Android 
*React-Native 0.59

**Comment by mykola-mokhnach** [comment]
Please fill out the required information in the issue reporting form
